src/system/connectivity/wifi.cpp

```diff
@@ -46,18 +47,18 @@ wl_status_t WiFiConnection::WifiInitialize (
     const char* wifi_password
     )
 {
-    String templ_id,dev_name, auth_token = "";
+    String chat_id,telegram_token, auth_token = "";
```

Comment:
Please, avoid this kind of assignment.
```suggestion
    String chat_id = "";
    String telegram_token = "";
    String auth_token = "";
```